common/app/views/mainAMP.scala.html

```diff
@@ -44,6 +44,38 @@
         <amp-pixel src="//www.facebook.com/tr?id=@FBPixel.account&ev=PageView&noscript=1"></amp-pixel>
         <amp-analytics config="https://ophan.theguardian.com/amp.json"></amp-analytics>
 
+        <amp-analytics type="googleanalytics" id="google-analytics">
+            <script type="application/json">
+            {
+              "requests": {
+                "pageviewWithCustomDims": "${pageview}&cd3=${platform}&cd4=${sectionId}&cd5=${contentType}&cd6=${commissioningDesks}&cd7=${contentId}&cd8=${contributorIds}&cd9=${keywordIds}&cd10=${toneIds}&cd11=${seriesId}"
+              },
+              "vars": {
+                "account": "@{GoogleAnalyticsAccount.editorialTracker.trackingId}"
+              },
+              "triggers": {
+                "trackPageview": {
+                  "on": "visible",
+                  "request": "pageviewWithCustomDims",
+                  "vars": {
+                    "title": "@{content.metadata.webTitle}",
+                    "platform": "AMP",
+                    "sectionId": "@{content.metadata.sectionId}",
+                    "contentType": "@{content.metadata.normalisedContentType}",
+                    "commissioningDesks": "@{content.tags.commissioningDesks.mkString(",")}",
+                    "contentId": "@{content.metadata.id}",
+                    "contributorIds": "@{content.tags.contributors.map(_.id).mkString(",")}",
+                    "keywordIds": "@{content.tags.keywords.map(_.id).mkString(",")}",
+                    "toneIds": "@{content.tags.tones.map(_.id).mkString(",")}",
+                    "seriesId": "@{content.tags.series.map(_.id).headOption.getOrElse("")}"
+                  }
+                }
+              }
+            }
+            </script>
+        </amp-analytics>
+        <amp-pixel src="//beacon.guim.co.uk/count/pvg.gif"></amp-pixel>
```

Comment:
It's for the [confidence graph](https://frontend.gutools.co.uk/analytics/confidence). 

The idea is that we download one pixel (`/count/pv.gif`) as soon as the page loads, and another pixel (`/count/pvg.gif`) after the GA request has successfully completed. That way we can measure how accurately GA is counting our pageviews. There is an equivalent pixel (`/count/pva.gif`) for Omniture as well.

At least that's how it works on normal pages - with AMP, there's no way to know when the GA call has successfully completed, so I had to just put the pixel directly after the `amp-analytics` tag.

Actually it looks like the initial `pv.gif` pixel is not being requested, so I'll add that as well.